workers.py

```diff
@@ -117,3 +117,75 @@ def worker() -> None:
 
     thread = Thread(target=worker)
     thread.start()
+
+
+def topology_client_download(topology_client, entity: str) -> list:
+    """Download a list for a Topological Inventory Entity."""
+    return topology_client.query_path(entity)
+
+
+def download_topological_inventory_data(
```

Comment:
If we want to introduce multiple worker types, we should do that via the worker targets. The lifecycle should be managed from one method. For now, I can live with this. but later we need to have this refactored. This is not a good solution.